depends/windows/mingw/03-use-curl.patch

```diff
@@ -0,0 +1,11 @@
+--- a/etc/pacman.conf	Fri Jul 15 02:58:59 2016
++++ b/etc/pacman.conf	Tue Oct 20 20:36:28 2020
+@@ -15,7 +15,7 @@
+ #LogFile     = /var/log/pacman.log
+ #GPGDir      = /etc/pacman.d/gnupg/
+ HoldPkg      = pacman
+-#XferCommand = /usr/bin/curl -C - -f %u > %o
++XferCommand = /usr/bin/curl -k -L -s -C - -f %u > %o
```

Comment:
I'd add --retry 5 instead of -k (turns off cert validation)

Reply:
In the end the only way to make this work reliably was to combine -k and — retry: `/usr/bin/curl -k --retry 5 -L -C - -f %u > %o`